ogc/src/main/scala/geotrellis/server/ogc/wcs/WcsParams.scala

```diff
@@ -53,20 +52,37 @@ case class DescribeCoverageWcsParams(version: String, identifiers: Seq[String])
  * Parameters descriptions can be also found here: https://mapserver.org/ogc/wcs_server.html
  */
 case class GetCoverageWcsParams(
-  version: String,
-  identifier: String,
-  boundingBox: Extent,
-  format: OutputFormat,
-  gridBaseCRS: CRS,
-  gridCS: URI,
-  gridType: URI,
-  // GridOrigin is BBOX minx, maxy // swapped in case of a geographic projection
-  gridOrigin: (Double, Double),
-  // GridOffsets is xres, yres // swapped in case of a geographic projection
-  gridOffsets: (Double, Double),
-  crs: CRS
+                                 version: String,
+                                 identifier: String,
+                                 boundingBox: Extent,
+                                 temporalSequence: Option[NonEmptyList[OgcTimeInterval]],
+                                 format: OutputFormat,
+                                 gridBaseCRS: CRS,
+                                 gridCS: URI,
+                                 gridType: URI,
+                                 // GridOrigin is BBOX minx, maxy // swapped in case of a geographic projection
+                                 gridOrigin: (Double, Double),
+                                 // GridOffsets is xres, yres // swapped in case of a geographic projection
+                                 gridOffsets: (Double, Double),
+                                 crs: CRS
```

Comment:
Do you think such alignment is more readble or you're using a default IDEA formatter?

Reply:
This was just a formatter choice. Should I revert?

Reply:
shrug